Dockerfile

```diff
@@ -14,6 +14,11 @@ RUN bin/bash -l -c "gem install aws-sdk mime-types"
 RUN apt-get install -y python2.7
 ENV PYTHON /usr/bin/python2.7
 
+# install hugo
+ENV HUGO_VERSION 0.21
```

Comment:
Here is our repo with just our HUGO code — https://github.com/GSA/digital.gov-hugo
Let @thisisdano or I know if you run into issues getting it started up